README.md

```diff
@@ -1,7 +1,42 @@
 # Community-Curated-Resources
+
 Community Curated Resources for development by members of Facebook Developer Circle Delhi, NCR. The repository is a collection of resources which are split according to technologies/languages/frameworks. Each topic has a MD file for it. The contents of a file are divided into three levels which are the level of expertise required to understand the resource mentioned (beginner, intermediate and expert).
 
+
+
+## Resources
+
+### [Angular](./Angular.md)
+
+### [C#](./CSharp.md)
+
+### [Golang](./Go.md)
+
+### [Javascript](./Javascript.md)
+
+### [ReactJS ](./ReactJS.md)
+
+### [Redux](./Redux.md)
+
+### [Python](./Python.md)
+
+### [Django - Python Framework](./Django.md)
+
+
+
 ## Contributing Guidelines
+
 * If you have a resource that you would like to add to the list, add it in the appropriate section of the file (for that specific topic).
-* If the file does not exist, copy the [template file](../master/TEMPLATE.md) and rename it to the topic that you wish to add the resource for and then submit a pull request.
+
+* If the file does not exist, copy the [template file](../master/TEMPLATE.md) and rename it to the topic that you wish to add the resource for and then submit a pull request.<br /> Here's the template for quick copy-paste:
+  ```markdown
+  # [Topic Name] Related Resources
+
+  ## For Beginners
+
+  ## For People with Intermediate Level Expertise
+
+  ## For Experts
+  ```
+
```

Comment:
The link to the template is sufficient. The template can evolve over time, duplicating it here will be a maintenance burden in the future.